setup.py

```diff
@@ -53,12 +53,14 @@
     install_requires=[
         "Django>=2.2",
         "requests>=1.2.0",
+        "django-phonenumber-field[phonenumberslite]>=6.0.0",
```

Comment:
Can you add a note to the changelog that this is now required? (for v0.16).

Oh, also mention the new provider in the changelog.

Reply:
You did, my bad. But it needs to be 0.16. It has conflicts now too, since I pushed dropping py36.

Reply:
https://github.com/jazzband/django-payments/pull/292/commits/7d624b43ffe3ac33b1671c2d2c870f65c8621dfb
I also added a reminder to make migrations since BasePayment is abstract.